waterloop-site/src/components/SustainableTech/Hero/Hero.tsx

```diff
@@ -1,5 +1,93 @@
 import React from 'react';
+import styled from 'styled-components';
 
-const Hero: React.FC = () => <div>Hero</div>;
+import LandingImage from '../../../static/img/sustainable-tech/HeroLandingImage.svg';
+import GreenTechLogo from '../../../static/img/sustainable-tech/GreenTechLogo.svg';
 
+const LandingContainer = styled.div`
+  padding: 60px 0 30px;
+  display: flex;
+  justify-content: center;
+  background-color: #e5f6fa;
+  overflow: hidden;
+`;
+
+const LandingContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow: hidden;
+`;
+
+const LandingImageContainer = styled.img`
+  height: 100vh;
```

Comment:
More than height, I think `width` is more important. Currently, the SVG doesn't fill up the entire width of the screen. I would replace this with `width: 100%`